src/postcapture.rs

```diff
@@ -43,6 +53,30 @@ impl PostCaptureAction for SaveAndCopy {
             Err(why) => tracing::error!("Failed to save screenshot to file: {why}"),
         }
 
+        if let Err(why) = db::add_screenshot_to_history(conn, Some(path.clone()), now.clone(), None)
+        {
+            tracing::error!("Failed to add screenshot to history: {why}");
+            return;
+        }
+        if let Err(why) = model_notifier.send(RowData::new_from_components(Some(path), now, None)) {
+            tracing::error!("Failed to notify the history model that a new item was added: {why}");
+        }
```

Comment:
I wonder if we should have an action for adding an item to history (not a blocker for this PR)